test/integration/targets/include_parent_role_vars/tasks/included_by_other_role.yml

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: ensure our parent role tree to contain only our direct parent item
+  assert:
+    that:
+      - "ansible_parent_role_names == ['special_vars']"
+
+- name: attempt to include ourselves
+  include_role:
```

Comment:
We need tests for `import_role` as well.

Reply:
Added.